libs/@local/hash-graphql-shared/src/graphql/type-defs/knowledge/entity.typedef.ts

```diff
@@ -39,6 +39,28 @@ export const entityTypedef = gql`
     linkedEntities: [LinkedEntityDefinition!]
   }
 
+  """
+  An entity proposed for creation. The suggested data can be used in further calls, e.g. to createEntity
+  """
+  type ProposedEntity {
+    """
+    The suggested type of the proposed entity.
+    """
+    entityTypeId: VersionedUrl!
```

Comment:
The entity type id is not going to be suggested. Entities are going to be suggested based on entity types.

Reply:
I would say it's still a suggested entityTypeId because a user may choose to assign a different type to an entity, where there are multiple valid types for it. But I've removed 'suggested' from this comment anyway